cloud/general/DOT-asgs-brew.sh

```diff
@@ -282,6 +282,16 @@ _reset_ephemeral_envars() {
   export ASGS_CONFIG=
 }
 
+# used to unset variables stored in statefile when
+# the statefile is not found or is moved or deleted
+_unset_statevars() {
+  export STATEFILE=
```

Comment:
Is there a reason that `unset` would not be used here? I do like the idea of cleaning things up like this; and I am not exactly sure of the side effect of `unset VAR` versus `export VAR=`. Did you try `unset` and find it was causing issues?

Reply:
I used `export VAR=` to keep this consistent with the existing implementation of `_reset_ephemeral_envars()`'  I thought about using `unset` instead but was not sure if the unsetting would propagate to a subshell, and it seemed that `export VAR=` definitely would. However, I just did a little command line test and found that the unsetting does seem to propagate. So I will change this from `export VAR=` to `unset` and test.